pkg/cmd/issue/edit/edit_test.go

```diff
@@ -351,38 +546,48 @@ func Test_editRun(t *testing.T) {
 		},
 	}
 	for _, tt := range tests {
-		ios, _, stdout, stderr := iostreams.Test()
-		ios.SetStdoutTTY(true)
-		ios.SetStdinTTY(true)
-		ios.SetStderrTTY(true)
+		t.Run(tt.name, func(t *testing.T) {
+			ios, _, stdout, stderr := iostreams.Test()
+			ios.SetStdoutTTY(true)
+			ios.SetStdinTTY(true)
+			ios.SetStderrTTY(true)
 
-		reg := &httpmock.Registry{}
-		defer reg.Verify(t)
-		tt.httpStubs(t, reg)
+			reg := &httpmock.Registry{}
+			defer reg.Verify(t)
+			tt.httpStubs(t, reg)
 
-		httpClient := func() (*http.Client, error) { return &http.Client{Transport: reg}, nil }
-		baseRepo := func() (ghrepo.Interface, error) { return ghrepo.New("OWNER", "REPO"), nil }
+			httpClient := func() (*http.Client, error) { return &http.Client{Transport: reg}, nil }
+			baseRepo := func() (ghrepo.Interface, error) { return ghrepo.New("OWNER", "REPO"), nil }
 
-		tt.input.IO = ios
-		tt.input.HttpClient = httpClient
-		tt.input.BaseRepo = baseRepo
+			tt.input.IO = ios
+			tt.input.HttpClient = httpClient
+			tt.input.BaseRepo = baseRepo
 
-		t.Run(tt.name, func(t *testing.T) {
 			err := editRun(tt.input)
-			assert.NoError(t, err)
+			if tt.wantErr {
+				assert.Error(t, err)
+			} else {
+				assert.NoError(t, err)
+			}
 			assert.Equal(t, tt.stdout, stdout.String())
-			assert.Equal(t, tt.stderr, stderr.String())
+			// Use regex match since mock errors and service errors will differ.
+			assert.Regexp(t, regexp.MustCompile(tt.stderr), stderr.String())
```

Comment:
```suggestion
			assert.Regexp(t, tt.stderr, stderr.String())
```
Looks like `assert.Regexp` will take care of the regexp compiling for us.